### PR TITLE
Update: TreePicker performance thru immutability

### PR DESCRIPTION
**Pull request by camjc**
- Fix: Bug where breadcrumbNodes were not
  reset on changeRootType.

### CODE REVIEW

File: test/helpers/deepSetObjectMutability.js [hunk 1]
```diff
@@ -1,14 +0,0 @@
-import _ from 'lodash';
```
Comment (etaoins) [inline test/helpers/deepSetObjectMutability.js]: :fire:

**Comment by etaoins** [comment]
LGTM!

**Comment by vinteo** [comment]
LGTM

**Comment by etaoins** [comment]
Merging :rocket:

**Pull request merged**